chainercv/links/connection/convolution_2d_block.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+
+import chainer
+from chainer.functions import relu
+from chainer.links import BatchNormalization
+from chainer.links import Convolution2D
+
+
+class Convolution2DBlock(chainer.Chain):
+    """Convolution2D --> (Batch Normalization) --> Activation
```

Comment:
DenseNet and some newer researches use `BN->ReLU->Conv` as a basic block. I guess it will be more common. We should not assume `Conv->BN->ReLU` is common in the future.